libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -277,4 +280,11 @@ public void onServiceDisconnected(ComponentName className) {
       serviceBound = false;
     }
   };
+
+  public String obtainAudioType() {
```

Comment:
Where are we using `obtainAudioType()`? `MapboxTelemetry` shouldn't expose this functionality. `obtainAudioType()` should live in another class in which has more context of what it's doing. Events?